parcels/codegenerator.py

```diff
@@ -352,9 +359,28 @@ def visit_Assign(self, node):
     def visit_Call(self, node):
         node.func = self.visit(node.func)
         node.args = [self.visit(a) for a in node.args]
+        node.keywords = {kw.arg: self.visit(kw.value) for kw in node.keywords}
+
         if isinstance(node.func, ParticleAttributeNode) \
            and node.func.attr == 'state':
             node = IntrinsicNode(node, "return DELETE")
+
+        if isinstance(node.func, FieldEvalCallNode):
+            # get a temporary value to assign result to
+            tmp = self.get_tmp()
+            # whether to convert
+            convert = True
+            if "applyConversion" in node.keywords:
+                k = node.keywords["applyConversion"]
+                if isinstance(k, ast.NameConstant):
```

Comment:
This could be handled more indirectly, adding an `if` statement to the `conv_stat` below, but I've just handled the static case for now.